src/main/java/com/angelbroking/smartapi/SmartConnect.java

```diff
@@ -804,6 +799,37 @@ public JSONObject logout() {
 			return null;
 		}
 	}
-    
+
+	/**
+	 * Get Margin Data.
+	 *
+	 * @param marginParams is margin data params.
+	 * @return returns the response of margin data.
+	 */
+	public JSONObject getMarginDetails(List<MarginParams> marginParams) {
+		try {
+			JSONArray positionsArray = new JSONArray();
+
+			for (MarginParams params : marginParams) {
+				JSONObject position = new JSONObject();
+				position.put("exchange", params.exchange!= null ? params.exchange : "");

```

Comment:
Remove null condition.

Reply:
Removed the null condition from method, please check